hoomd/mpcd/RejectionVirtualParticleFiller.h

```diff
@@ -0,0 +1,152 @@
+
+#ifndef MPCD_REJECTION_FILLER_H_
```

Comment:
Include guards need to match the file name:
```suggestion
#ifndef MPCD_REJECTION_VIRTUAL_PARTICLE_FILLER_H_
```